node/collation-generation/src/lib.rs

```diff
@@ -146,7 +146,11 @@ impl CollationGenerationSubsystem {
 			}
 			Ok(Signal(BlockFinalized(_))) => false,
 			Err(err) => {
-				log::error!(target: "collation_generation", "error receiving message from subsystem context: {:?}", err);
+				log::error!(
+					target: "collation_generation",
```

Comment:
while we're at it, would be nice to extract it into a const `LOG_TARGET` as in other crates

Reply:
Good call! https://github.com/paritytech/polkadot/pull/1753/commits/2ce6f18aa04ff106a1a386a4e5d4661c95b55546